lte/gateway/python/magma/pipelined/rpc_servicer.py

```diff
@@ -10,12 +10,16 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 """
+# pylint: disable=unsubscriptable-object
```

Comment:
Context: This is a bug and will not be needed for future versions of pylint.
https://github.com/PyCQA/pylint/issues/6523

Reply:
The fix in the new version of pylint is just to disable the check in all files that do `from __future__ important annotations`. So the line will have no effect and we won't need it any more but I'm not sure it's really worth tracking that.